README.md

```diff
@@ -13,6 +13,23 @@ As this package is just a wrapper around `flint`, you will need something to
 actually run it. By default the [Linter][] package will be installed for you to
 fill this need.
 
+# Considerations
```

Comment:
```suggestion
## Considerations
```
Only the title should ever be at the `#` level. The `# Installation` line should also be fixed.